pkg/commands/commands.go

```diff
@@ -340,6 +344,37 @@ func GolangCI(fix bool) error {
 	)
 }
 
+
+func ReviewDog(PR int) error {
+	// userName, err := GetUserName()
+	// if err != nil {
+	// 	return err
+	// }
+
+	gitOrg, gitRepo, err := GetModules()
+	if err != nil {
+		return err
+	}
+
+	command := []string{
+		fmt.Sprintf("export CI_PULL_REQUEST=%d;", PR),
+		fmt.Sprintf("export CI_REPO_OWNER=%s;", gitOrg),
+		fmt.Sprintf("export CI_REPO_NAME=%s;", gitRepo),
+		fmt.Sprintf("export CI_COMMIT=$(git rev-parse HEAD);"),
+		"golint ./... | reviewdog -f=golint -diff=\"git diff FETCH_HEAD\" -reporter=github-pr-review",
+	}
+
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>unnecessary use of fmt.Sprintf (S1039)